### PR TITLE
Linkedin,Facebook links of Mattermost Updated in  README.md

### PR DESCRIPTION
**Pull request by drraghavendra**
Linkedin,Facebook links of Mattermost Updated in  README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ Learn More:
 
 Get the Latest News:
 
+- **facebook** - Follow [Mattermost](https://www.facebook.com/MattermostHQ)
+- **linkedin** - Follow [Mattermost](https://www.linkedin.com/company/mattermost/)
```
Comment (hanzei) [inline README.md:48]: Would you please keep Twitter and Email as the first two items?

File: plugins/pluggable/Pluggable.jsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
+import PropTypes from 'prop-types';
```
Comment (hanzei) [inline plugins/pluggable/Pluggable.jsx:1]: That seems like an unrelated  change

**Review by hanzei — CHANGES_REQUESTED** [review]
Thanks for helping with the README

File: README.md [hunk 3]
```diff
@@ -44,6 +44,8 @@ Learn More:
 
 Get the Latest News:
 
+- **facebook** - Follow [Mattermost](https://www.facebook.com/MattermostHQ)
```
Comment (hanzei) [inline README.md:47]: Could you also please sync with https://github.com/mattermost/mattermost-server#get-the-latest-news?

**Comment by drraghavendra** [comment]
Please be informed Linkedin,Facebook links are provided.So i request you to review and intimate me for any necessary updates to be made

File: README.md (hunk 3 above)
Comment (drraghavendra) [reply README.md:47]: Sir updated https://github.com/mattermost/mattermost-server/pull/16186

**Comment by jasonblais** [comment]
Hi @drraghavendra, looks like @hanzei shared feedback above. Would you be open to keep Twitter and Email as the first two items, and remove the unrelated changes to `Pluggable.jsx`?

Let me know if you have any questions about them :)

**Comment by drraghavendra** [comment]
Sir @jasonblais please make a note as this PR is related to Linkedin,Facebook links only.

**Comment by jasonblais** [comment]
@drraghavendra thank you again for the contribution. Would you be open to help with two changes?

1 - Remove the changes to `Pluggable.jsx` in this pull request, given you submitted them separately in mattermost/mattermost-server#16186
2 - Help capitalize Facebook and LinkedIn as `Facebook`, `LinkedIn` respectively, and move them below the Twitter and Email options on the list?

If you need any help making the changes, let us know. Thanks again!

**Comment by jfrerich** [comment]
Hi, @drraghavendra, if you could please address the remaining PR feedback, we'd love to merge this request.  Thanks!

**Comment by drraghavendra** [comment]
please update on the above Pull request as the code is viable and updated

**Comment by drraghavendra** [comment]
Sir Request you to please update on the above Pull request as the code is viable @jasonblais @jfrerich @emilyacook

**Comment by jasonblais** [comment]
@drraghavendra sorry for the delay - looks like the LinkedIn and Facebook links were updated via your other PR at https://github.com/mattermost/mattermost-server/pull/16186

I don't see other related changes here, are you open to closing this in favor of https://github.com/mattermost/mattermost-server/pull/16186?

**Comment by drraghavendra** [comment]
Truly agree and Admit as the above PR same as  https://github.com/mattermost/mattermost-server/pull/16186 .Hence Request you to merge.

**Comment by jasonblais** [comment]
@drraghavendra thank you! Given the changes in this PR are the same as https://github.com/mattermost/mattermost-server/pull/16186, I'll close this PR as a duplicate.
